### DISCUSSION TITLE
Functionality of sorting by latest update and filtering by version

### DISCUSSION THREAD

**Discussion by justastranger** [modrinth/.github#11] [Feature requests]
### Is your feature request related to a problem? Please describe.

Using the sorting and filtering mentioned in the title, the mods are arranged by the latest update, regardless of what Minecraft version it's for.

### Describe the solution you'd like

When sorting and filtering in the way mentioned in the title, only sort based on the latest update *for* the filtered version of Minecraft.

### Describe alternatives you've considered

Praying to find the updates I'm looking for for my 1.16.5 modpack that I haven't updated since 2021.

**Comment by triphora** [comment]
I'm not entirely sure I understand your request. That said, the MC version filters in search and on individual projects' `Versions` tab should cover this use-case.

**Comment by justastranger** [comment]
That use-case is only useful when you actually get to the mod page. This would be the step that helps you get to the relevant mod pages.

As it stands, you have no idea if the date for the mod's latest update is for the version you're filtering for and therefore no way to know when the latest update for that version of Minecraft actually happened without digging through one or two more layers of menus past the search results.
